code/modules/power/tesla/energy_ball.dm

```diff
@@ -1,5 +1,5 @@
-#define TESLA_DEFAULT_POWER 3476520
-#define TESLA_MINI_POWER 1738260
+#define TESLA_DEFAULT_POWER 1738260
```

Comment:
because once you account for all the coils, it ends up putting out 4 times more than the smses can even take in.

Honestly it's not far enough, but it's something.